### PR TITLE
Fix Release Notes formatting

### PR DESCRIPTION
**Pull request by julioest**
### Summary
This PR addresses #932 and #1083

### CODE REVIEW

File: core/htmlhelper.py [hunk 1]
```diff
@@ -411,6 +411,48 @@ def style_links(soup, class_name):
...
+            existing_classes = target.get("class", [])
+            if class_to_add not in existing_classes:
+                target["class"] = existing_classes + [class_to_add]
+                return target
+    return None
+
+
+def reformat_new_libraries_list(soup):
+    new_libraries_list = add_class_to_sibling_by_header(
+        soup, "New Libraries", "new-libraries", search_tag="h3", target_tag="ul"
```
Comment (kennethreitz) [inline core/htmlhelper.py:434]: just to make it clearer for the future, it'd be better to pass the argument names here (e.g. `class_to_add="new-libraries" ...)`

**Review by kennethreitz — CHANGES_REQUESTED** [review]

File: core/htmlhelper.py (hunk 1 above)
Comment (julioest) [reply core/htmlhelper.py]: Gotcha, makes sense! Thanks

Comment (kennethreitz) [reply core/htmlhelper.py]: also, `return None` happens automatically if nothing else is returned :)

File: core/htmlhelper.py [hunk 2]
```diff
@@ -411,6 +411,52 @@ def style_links(soup, class_name):
...
+    """
+    sections = soup.find_all(header_tag, string=header_text)
+    for section in sections:
+        target = section.find_next_sibling(target_tag)
+        if target:
+            existing_classes = target.get("class", [])
+            if class_to_add not in existing_classes:
+                target["class"] = existing_classes + [class_to_add]
+                return target
+    return None
```
Comment (kennethreitz) [inline core/htmlhelper.py:429]: so just nix this part

Comment (julioest) [reply core/htmlhelper.py]: Removed line 429

File: core/htmlhelper.py (hunk 1 above)
Comment (julioest) [reply core/htmlhelper.py]: TY, Lord Kenneth. Removed statement in the latest commit

**Comment by kennethreitz** [comment]
Excellent! ✨  🍰  ✨

**Comment by kennethreitz** [comment]
@sdarwin can you run `python manage.py import_release_notes` against staging for us?

**Comment by sdarwin** [comment]
@kennethreitz, done. If the following new text is found in the release notes "* The download links are supported by grants from [The C++ Alliance](https://cppalliance.org/). "  it should be removed, in the same way the actual download links were, since the message goes with the actual downloads.

**Pull request merged**